src/common/utility/BJHash.h

```diff
@@ -95,20 +96,22 @@ inline unsigned int BJHash::Hash(const unsigned char *k, unsigned int length, un
 
    c += length;
 
-   switch(len)
-   {
-      case 11: c+=((unsigned int)k[10]<<24);
-      case 10: c+=((unsigned int)k[9]<<16);
-      case 9 : c+=((unsigned int)k[8]<<8);
-      case 8 : b+=((unsigned int)k[7]<<24);
-      case 7 : b+=((unsigned int)k[6]<<16);
-      case 6 : b+=((unsigned int)k[5]<<8);
-      case 5 : b+=k[4];
-      case 4 : a+=((unsigned int)k[3]<<24);
-      case 3 : a+=((unsigned int)k[2]<<16);
-      case 2 : a+=((unsigned int)k[1]<<8);
-      case 1 : a+=k[0];
-   }
+   std::vector<std::function<void()>> cases = {
```

Comment:
Looks like it does the same thing as previously, have you verified?

Reply:
I like this a lot.

Reply:
Yes, I have now verified using a bunch of VisIt source code files as strings to input. Both produce the same values now.